src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -241,8 +241,19 @@ class JooqBurdenEstimateRepository(
         return parameterSetId
     }
 
-    override fun getModelRunParameterSet(setId: Int): FlexibleDataTable<ModelRun>
+    override fun getModelRunParameterSet(groupId: String, touchstoneVersionId: String, setId: Int): FlexibleDataTable<ModelRun>
     {
+        //First check if this set id actually belongs to this group and touchstone version
+        dsl.select(MODEL_RUN_PARAMETER_SET.ID)
+                .from(MODEL_RUN_PARAMETER_SET)
+                .join(RESPONSIBILITY_SET)
+                .on(RESPONSIBILITY_SET.ID.eq(MODEL_RUN_PARAMETER_SET.RESPONSIBILITY_SET))
```

Comment:
I think you can use our util `fromJoinPath` here, it saves you having to explicitly provide the join key when it's unambiguous. e.g.
```
  dsl.select(MODEL_RUN_PARAMETER_SET.ID)
                .fromJoinPath(MODEL_RUN_PARAMETER_SET, RESPONSIBILITY_SET)
                .where(...)
```